[2024-12-11T21:19:46.830+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-11T21:19:46.847+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-08-16T00:00:00+00:00 [queued]>
[2024-12-11T21:19:46.858+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-08-16T00:00:00+00:00 [queued]>
[2024-12-11T21:19:46.858+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-11T21:19:46.874+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_book_data> on 2024-08-16 00:00:00+00:00
[2024-12-11T21:19:46.883+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=895) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-11T21:19:46.884+0000] {standard_task_runner.py:72} INFO - Started process 909 to run task
[2024-12-11T21:19:46.885+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_store_google_books', 'fetch_book_data', 'scheduled__2024-08-16T00:00:00+00:00', '--job-id', '841', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp_meodpeq']
[2024-12-11T21:19:46.886+0000] {standard_task_runner.py:105} INFO - Job 841: Subtask fetch_book_data
[2024-12-11T21:19:46.933+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-08-16T00:00:00+00:00 [running]> on host bb74d1700244
[2024-12-11T21:19:47.019+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_store_google_books' AIRFLOW_CTX_TASK_ID='fetch_book_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-16T00:00:00+00:00'
[2024-12-11T21:19:47.020+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-11T21:19:47.021+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-11T21:19:47.021+0000] {logging_mixin.py:190} INFO - Current task name:fetch_book_data state:running start_date:2024-12-11 21:19:46.848271+00:00
[2024-12-11T21:19:47.021+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_google_books and current dag run status:running
[2024-12-11T21:19:47.022+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-11T21:19:51.487+0000] {logging_mixin.py:190} INFO - Failed to fetch data. Status code: 429
[2024-12-11T21:19:51.506+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-11T21:19:51.518+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-11T21:19:51.518+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_google_books, task_id=fetch_book_data, run_id=scheduled__2024-08-16T00:00:00+00:00, execution_date=20240816T000000, start_date=20241211T211946, end_date=20241211T211951
[2024-12-11T21:19:51.531+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-11T21:19:51.532+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-11T21:19:51.532+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_google_books queued_at:2024-12-11 21:19:44.526922+00:00
[2024-12-11T21:19:51.532+0000] {logging_mixin.py:190} INFO - Task hostname:bb74d1700244 operator:PythonOperator
[2024-12-11T21:19:51.588+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-11T21:19:51.613+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-11T21:19:51.615+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
